programs/mpl-core/src/processor/collect.rs

```diff
@@ -75,10 +95,19 @@ fn collect_from_account(account_info: &AccountInfo, dest_info: &AccountInfo) ->
         _ => return Err(MplCoreError::IncorrectAccount.into()),
     };
 
-    let dest_starting_lamports = dest_info.lamports();
-    **dest_info.lamports.borrow_mut() = dest_starting_lamports
-        .checked_add(fee_amount)
+    let split_fee_amount = fee_amount
+        .checked_div(2)
+        .ok_or(MplCoreError::NumericalOverflowError)?;
```

Comment:
I think technically you can lose a lamport here if the lamports are odd but I think that is likely never the case.